src/main/java/net/mcreator/ui/modgui/BiomeGUI.java

```diff
@@ -517,51 +539,29 @@ public BiomeGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		return new AggregatedValidationResult.PASS();
 	}
 
-	private void updateBiomeTreesForm() {
-		if (customTrees.isSelected()) {
-			vanillaTreeType.setEnabled(false);
-			minHeight.setEnabled(true);
-			spawnVines.setEnabled(true);
-			treeVines.setEnabled(true);
-			treeStem.setEnabled(true);
-			treeBranch.setEnabled(true);
-			treeFruits.setEnabled(true);
-		} else {
-			vanillaTreeType.setEnabled(true);
-			minHeight.setEnabled(false);
-			spawnVines.setEnabled(false);
-			treeVines.setEnabled(false);
-			treeStem.setEnabled(false);
-			treeBranch.setEnabled(false);
-			treeFruits.setEnabled(false);
-		}
-	}
-
 	@Override public void openInEditingMode(Biome biome) {
 		name.setText(biome.name);
 		groundBlock.setBlock(biome.groundBlock);
 		undergroundBlock.setBlock(biome.undergroundBlock);
-		treeVines.setBlock(biome.treeVines);
-		treeStem.setBlock(biome.treeStem);
-		treeBranch.setBlock(biome.treeBranch);
-		treeFruits.setBlock(biome.treeFruits);
-
-		if (biome.treeType == biome.TREES_CUSTOM) {
-			vanillaTrees.setSelected(false);
-			customTrees.setSelected(true);
-		} else {
-			vanillaTrees.setSelected(true);
-			customTrees.setSelected(false);
-		}
-
-		spawnVines.setSelected(biome.spawnVines);
-		minHeight.setValue(biome.minHeight);
 		airColor.setColor(biome.airColor);
 		grassColor.setColor(biome.grassColor);
 		foliageColor.setColor(biome.foliageColor);
 		waterColor.setColor(biome.waterColor);
 		waterFogColor.setColor(biome.waterFogColor);
-		treesPerChunk.setValue(biome.treesPerChunk);
+
+		if (biome.customTree) {

```

Comment:
Set fields in either case, even if not enabled so UI state is preserved in mod element.